website/docs/advanced-concepts/granular-access-control/roles.md

```diff
@@ -1,77 +1,72 @@
+</div>
 
-- **Instance Administrator**: This role provides a user with permission to modify settings at the instance level from the Admin settings. This includes changing the general settings of the instance, such as authentication, email, custom branding, access to view audit logs and granular access control actions. It's important to note that this role has significant control over the instance, so it should only be assigned to trusted users who need these capabilities.
+<!-- vale on -->
 
-## Custom roles
+To configure Granular Access Control (GAC), Appsmith provides pre-defined roles along with the flexibility to create your own roles. This page provides information on default and custom roles in Appsmith.
 
-With custom roles, you can provide fine-grained access control by configuring multiple permissions for the role you are creating. To create a custom role, go to **Admin Settings** > **Access Control** > **Roles** and click the **Add Role** button.
+## Default roles
 
-<ZoomImage
-  src="/img/add_custom_role.png" 
-  alt="Add a new Role"
-  caption="Add a custom role"
-/>
+Default roles provide standardized permission sets designed to match different user responsibilities in Appsmith. They include pre-defined permissions tailored for roles such as administrators, developers, and app viewers, and cannot be altered or deleted. You can assign these roles to your users if the permissions align with your desired permission model. To view the default roles available for your Appsmith instance, toggle the **Default Roles**  option on the Roles screen.
 
-The permissions are grouped into four categories, which helps users manage and access the necessary permissions easily:
+ <ZoomImage
+    src="/img/GAC-Default-Roles-toggle.png" 
+    alt="Toggle the Default roles options to view Default Roles"
+    caption="Toggle the Default roles options to view Default Roles"
+  />
 
-- [Application Resources](#application-resources)
-- [Datasource and Environments](#datasource-and-environments)
-- [Groups and Roles](#groups-and-roles)
-- [Others](#others)
 
+### Instance level
 
-### Application resources
+Instance-level default roles have permissions that govern access to Appsmith instance, affecting all workspaces and users within the instance. Some instance-level roles can be tailored to provide specific access to all users. The instance-level roles include:
 
-In this section, you'll find permissions related to the application and its resources such as pages, widgets, and queries. The table below illustrates the permissions that can be assigned to either a user or a group. Additionally, it explains how each permission behaves at various hierarchical levels.
+* [Instance Administrator role](/advanced-concepts/granular-access-control/reference/default-roles#instance-administrator-role) - to configure instances, manage user groups and roles, create workspaces, and monitor audit logs.
```

Comment:
There should be no execute in this image. You are showing the Groups & Roles tab of GAC which doesn't contain execute.
![image](https://github.com/appsmithorg/appsmith-docs/assets/16632903/9e99e92b-111c-405f-84b9-c96682035758)